charts/sn-platform/templates/prometheus/prometheus-configmap.yaml

```diff
@@ -79,10 +79,14 @@ data:
       - source_labels: [__meta_kubernetes_pod_label_job_name]
         action: drop
         regex: ".+"
-      # ignore prometheus,grafana,toolset
+      # ignore istio
+      - source_labels: [__meta_kubernetes_pod_label_istio]
```

Comment:
Is these Pods' metrics annotation generated by Istio metric merging?

Reply:
From the Prometheus page, it shows like 
<img width="1877" alt="截屏2022-07-07 下午9 30 55" src="https://user-images.githubusercontent.com/10498732/177785909-c4fb55e6-7824-4f77-bf04-9a1f26d7ab1a.png">
<img width="1881" alt="截屏2022-07-07 下午9 31 04" src="https://user-images.githubusercontent.com/10498732/177786111-fec8ae78-f7d6-4c65-9a1c-62f0405cb2bc.png">

Reply:
Can we make this scrape job only discover Pod in `.Values.namespace` to avoid unnecessary targets? because these targets won't be scraped successfully without extra configuration in Istio mode.

Reply:
is there a label called `istio`?

Reply:
@ciiiii @MarvinCai  Discover Pod in `{{ template "pulsar.namespace" . }}` should be okay, i will update my PR.

Reply:
PTAL @ https://github.com/streamnative/charts/pull/792/commits/c1d8e9322ac6e77a025776889ccfa46199ab8649